apache-maven/src/assembly/shared/mvnvalidate.ps1

```diff
@@ -0,0 +1,9 @@
+# check mvn home
+if (-not $env:MAVEN_HOME) {
+    $env:MAVEN_HOME = (Get-Item $PSScriptRoot"\..")
+}
```

Comment:
This is wrong. `MAVEN_HOME` is neither an environment variable, nor can it be passed externally. It is solely private. See https://github.com/apache/maven/blob/29283a10596c92a9283d789d4c661dd242cec916/apache-maven/src/assembly/shared/mvnvalidate

Reply:
Interesting... I assumed it had the similar interpretation as JAVA_HOME (which is an environment variable). Always dangerous, making assumptions... 🤔 

Thanks for pointing it out! I'll update the pull request.

Reply:
These scripts are implementation details and can change from version to version therefore we cannot share those between other Maven homes reliably.